network-rfc/11_tokenomics_v2.1.md

```diff
@@ -0,0 +1,78 @@
+- Removing the treasury-initiated buyback-and-burn mechanics which makes SQD a security
+- Moving the reward token out of scope
+- Introducing the fee switch to the Portals (to be activated in the future if necessary)
+- Making it possible to register Portals on EVMs (in particular, Base) and Solana. For Solana users, it opens up the posibility to pay in USDC or SOL.
+
+
+## The SQD Flows
+
+**Workers** 
+Workers serve the data and receive rewards in SQD. The reward depend on the number of served queries, the amount of delegated tokens and the uptime of the worker. The worker has to lock 100k SQD to participate in the network. The maximal amount of rewards distributed per worker and the delegators is controlled by a single parameter called `TARGET_APR`. The reward is then split abetween the worker and the delegators for the worker.  
+
+**Delegators**
+Delegators delegate SQD to workers to get a part of the worker reward. Both the amount of delegated tokens and the served queries affect the reward per query served. 
+
+**Data consumers**
+Data consumers query the network p2p using a Portal. The maximal bandwidth that can be consumed by a Portal is determined by the amount of SQD locked in the Portal contract. Thus, the data consumer either buys SQD on the open market and locks the desired amount themselves, or makes an _SQD Provision Offer_ to SQD holders willing to provide SQD in return to the fee. 
+
+An SQD Provision Offer is an agreement to lock SQD for a specified amount of time for a fee, paid as continuosly during the whole lock period. The fee is locked by the consumer in advance and can be paid in any of the supported tokens. Special conditions apply for extending the provision offer and withdrwals. 
+
+**SQD providers**
+SQD providers hold SQD and fullfill the matching _SQD Provision Offers_. Active providers can advertise their target fees in advance to make the market and set the expectations for the data consumers. 
+
+## Emission reduction
+
+The `TARGET_APR` currently set to 25% yearly, will be gradually reduced and replaced by the fees collected from the portals. 
+
+## Portal Payments
+
+The are two options to get data through the portals:
+- Lock SQD tokens (the existing flows)
+- Pay a subscription fee in one of the supported tokens, so that the SQD is locked by one or multiple SQD providers. 
+
+**The subscription flow**
+
+The user specifies:
+- the required bandwidth (which translates into the required SQD to be locked)
+- the terms (fix-term or auto-extension)
+- the price (the dApp will provide the current quotes of the SQD providers to give a reasonable offer)
+
+The user:
+- creates a Portal Registration Contract
+- makes the fee deposits
+- the willing SQD providers lock the SQD for the required term
+
+The fee is deducted every epoch and automatically split:
+- 50% to the SQD providers
+- 45% to the worker reward pool
+- 5% gets burnt
```

Comment:
Why?

Reply:
Probably the easiest would be then do implement this as follows:
1) 50% of the fees (in stables) go to the yield for SQD providers
2) 45% of the fees (in stables) go buying back SQD from the market
3) 5% of the fees (in stables) go to buying ETH
4) ETH and SQD from step 2) are added to the liquidity pool in equal proportiions
5) The rest of SQD from the step 2) go to the worker reward pool

Reply:
Agreed with DZ that the simplest option will be to just forward all the collected fees to the treasury to later be distributed according to these shares